apps/notification-producer/src/producers/TradeNotificationProducer.ts

```diff
@@ -7,10 +10,25 @@ import { doForever } from '../utils';
 import { Runnable } from '../../types';
 import { SubscriptionRepository } from '../repositories/SubscriptionsRepository';
 import { NotificationsIndexerStateRepository } from '../repositories/NotificationsIndexerStateRepository';
+import { getAddress, parseAbi } from 'viem';
+
+const EVENTS = parseAbi([
+  'event OrderInvalidated(address indexed owner, bytes orderUid)',
```

Comment:
BTW, this is tracked in the backend DB as well.
There's a table called `onchain_order_invalidations`.

<img width="242" alt="image" src="https://github.com/user-attachments/assets/4be72b38-de1b-4cc4-939c-f11544fe76c0" />

So in theory we don't need to watch for those events.